recipes/magics-metview/build.sh

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env bash
+
+set -e
+
+export PYTHON="$PYTHON"
+export PYTHON_LDFLAGS="$PREFIX/lib"
+export LDFLAGS="$LDFLAGS -L$PREFIX/lib -Wl,-rpath,$PREFIX/lib"
+export CFLAGS="$CFLAGS -fPIC -I$PREFIX/include"
+
+export REPLACE_TPL_ABSOLUTE_PATHS=0
+if [[ $(uname) == Linux ]]; then
+  export REPLACE_TPL_ABSOLUTE_PATHS=1
+fi
+
+mkdir ../build && cd ../build
+
+cmake -DCMAKE_INSTALL_PREFIX=$PREFIX \
+      -DENABLE_FORTRAN=0 \
+      -DENABLE_NETCDF=1 \
+      -DENABLE_METVIEW=1 \
+      -DREPLACE_TPL_ABSOLUTE_PATHS=$REPLACE_TPL_ABSOLUTE_PATHS \
+      $SRC_DIR
+
+make -j $CPU_COUNT
+
+if [[ $(uname) == Linux ]]; then
+    # Tell Linux where to find libGL.so.1 and other libs needed for Qt
+    export LD_LIBRARY_PATH=$LD_LIBRARY_PATH:$BUILD_PREFIX/x86_64-conda_cos6-linux-gnu/sysroot/usr/lib64/
```

Comment:
See https://conda-forge.org/docs/maintainer/knowledge_base.html#libgl for how to avoid this.

Reply:
Solid. Thanks!